contribute/contribute.rst

```diff
@@ -320,6 +327,23 @@ Be sure to check that the resulting build behaves as expected over:
 In (1.) above a db wipe is initiated helping to test the self-start code capability.
 See the initrock script and it's systemd trigger service: rockstor-init.
 
+We also have **automated tests** in place that cover our API's and core critical path functionality.
+It is expected that any changes to critical path code e.g. fs management / updates / Web-UI / services;
```

Comment:
Would a comma be preferred here?
```suggestion
It is expected that any changes to critical path code e.g. fs management / updates / Web-UI / services,
```

Reply:
I can go with that; but I do like my semi-colons.